docs/_includes/date-picker.html

```diff
@@ -0,0 +1,212 @@
+<div id="date-picker">
+    {% include section-header.html name="date-picker" version=page.versions.date-picker %}
+
+    <p>The date picker component is used to display a calendar from which dates can be selected, either as single items or as a range.</p>
+
+    <h3>Single Pane</h3>
+
+    <button class="btn" onclick="document.getElementById('date-picker-example').toggleAttribute('hidden')">Show/Hide</button>
+
+    <div id="date-picker-example" class="date-picker date-picker--single-month" role="dialog" aria-label="date picker">
+        <div class="date-picker__header">
+            <button
+                class="icon-btn icon-btn--small icon-btn--transparent"
+                aria-label="Show July 2019"
+                variant="icon"
+            >
+                <svg aria-hidden="true" class="icon icon--chevron-left" focusable="false">
+                    {% include symbol.html name="chevron-left" %}
+                </svg>
+            </button>
+            <h3>August 2019</h3>
+            <button
+                class="icon-btn icon-btn--small icon-btn--transparent"
+                aria-label="Show July 2019"
+                variant="icon"
+            >
+                <svg aria-hidden="true" class="icon icon--chevron-right" focusable="false">
+                    {% include symbol.html name="chevron-right" %}
+                </svg>
+            </button>
+        </div>
+        <div class="date-picker__body">
+            <table class="calendar">
```

Comment:
Seems like if we are creating a new `.calendar` block, it should have its own module and examples section...?

Reply:
Yep. This is why I had hesitations about having the higher-level wrapper be named `date-picker`. There can be instances where the calendar is used for other things besides picking dates, even just statically as a calendar so users can get a quick glance at a calendar on a page. While it makes sense for the controls to be outside the months and headers, it feels a bit odd for there to be such a big difference in action implied by the naming convention.

Reply:
We can still have that. The idea is that a calendar can be nested inside of a date-picker.

> There can be instances where the calendar is used for other things besides picking dates, even just statically as a calendar so users can get a quick glance at a calendar on a page

This would be the `.calendar` class. It is the `.date-picker` class that add the interactivity.

> it feels a bit odd for there to be such a big difference in action implied by the naming convention.

Yeah, I see what you mean, maybe we can discuss this more if we don't think the implications of this naming convention is strong enough? It's a valid point.